nemo/collections/nlp/models/language_modeling/megatron/bert_model.py

```diff
@@ -220,18 +238,29 @@ def __init__(
                 parallel_output,
                 openai_gelu,
                 onnx_safe,
+                sequence_parallel,
             )
             self._lm_head_key = 'lm_head'
             self.binary_head = None
             if self.add_binary_head:
+                """
+                Should we do something here when sequence parallel is enabled ? 
```

Comment:
The pooler layer is doing the gather? so there is no more SP ? please verify and remove the comments

Reply:
Yes thats true. Checked. Will remove